Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
+        go run github.com/air-verse/air@v1.61.1 \
+        --build.cmd "go build -o tmp/bin/main" --build.bin "tmp/bin/main" --build.delay "100" \
+        --build.exclude_dir "node_modules" \
+        --build.include_ext "go" \
+        --misc.clean_on_exit true
+    requires:
+      vars: [GITHUB_TOKEN, OPENAI_API_KEY]
+  test:
```

Comment:
The 'i18n-extract' and 'i18n-merge' tasks don't handle potential errors that could arise during the execution of the commands. It might be beneficial to implement error checks to ensure that the commands succeed before proceeding with subsequent tasks if they are dependent.